west.yml

```diff
@@ -62,7 +62,7 @@ manifest:
       revision: fa481784b3c49780f18d50bafe00390ccb62b2ec
       path: modules/hal/st
     - name: hal_stm32
-      revision: dee4413253623ec575bb8b58abd56f91afe903bb
+      revision: pull/47/head
```

Comment:
@FRASTM, https://github.com/zephyrproject-rtos/hal_stm32/pull/47 is now merged.
Can you update the SHA1 ?